composeApp/src/commonMain/kotlin/domain/Event.kt

```diff
@@ -0,0 +1,19 @@
+package domain
+
+data class Event (
+    val id: String,
+    val title: String,
+    val description: String,
```

Comment:
Have you used String instead of Date for a particular reason? If there's no Date class in the Multiplatform module, you can add the official `Kotlin-datetime` library (https://github.com/Kotlin/kotlinx-datetime)

Reply:
done